applications/asset_tracker_v2/src/cloud/cloud_codec/aws_iot/json_protocol_names_aws_iot.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#define MODEM_CURRENT_BAND     "band"
+#define MODEM_NETWORK_MODE     "nw"
+#define MODEM_ICCID	       "iccid"
+#define MODEM_FIRMWARE_VERSION "modV"
+#define MODEM_BOARD	       "brdV"
+#define MODEM_APP_VERSION      "appV"
+#define MODEM_RSRP	       "rsrp"
+#define MODEM_AREA_CODE	       "area"
+#define MODEM_MCCMNC	       "mccmnc"
+#define MODEM_CELL_ID	       "cell"
+#define MODEM_IP_ADDRESS       "ip"
+
+#define CONFIG_DEVICE_MODE		  "act"
+#define CONFIG_ACTIVE_TIMEOUT		  "actwt"
+#define CONFIG_MOVE_TIMEOUT		  "mvt"
+#define CONFIG_MOVE_RES			  "mvres"
+#define CONFIG_GPS_TIMEOUT		  "gpst"
+#define CONFIG_ACC_THRESHOLD		  "acct"
+#define CONFIG_NO_DATA_LIST		  "nod"
+#define CONFIG_NO_DATA_LIST_GNSS	  "gnss"
+#define CONFIG_NO_DATA_LIST_NEIGHBOR_CELL "ncell"
+
+#define DATA_VALUE     "v"
```

Comment:
Alignment of the define values look off in the PR, but that could be GitHub artifact